src/plugins/newsfeed/public/lib/never_fetch_driver.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { PublicMethodsOf } from '@kbn/utility-types';
+import type { Observable } from 'rxjs';
+import type { NewsfeedApiDriver } from './driver';
+import type { FetchResult } from '../types';
+
+/**
+ * NewsfeedApiDriver variant that never fetches results. This is useful for instances where Kibana is started
+ * without any user interaction like when generating a PDF or PNG report.
+ */
+export class NeverFetchNewsfeedApiDriver implements PublicMethodsOf<NewsfeedApiDriver> {
```

Comment:
NIT: extract a proper interface instead of having one implementation implementing a reflect of the other one.